docs/arc-iac-docs/reference_architectures/how_arc_iac_works.md

```diff
@@ -0,0 +1,203 @@
+This purpose of this page is to explain the additional functionality of ARC Infrastructure as Code (IaC) post Landing Zone Accelerator (LZA) creation.  
+
+## ARC Infrastructure as Code (IaC)
+We are deeply committed to the OpenSource community and actively contribute to various projects. 
+All the referenced resources, including our Terraform modules and configurations, can be found on our GitHub repository. 
+Our mission is to leverage existing solutions, refine them to meet our specific requirements, and collaborate within the organization to drive innovation without reinventing the wheel.
+
+These tools are not intended to serve as a singular solution, but rather to evolve continually. 
+Our modules and resources play a pivotal role in expediting the critical cloud infrastructure, ensuring accelerated cloud adoption for our clients.
+
+### ARC IaC Module Reference
+**Referencing the Modules**:
+* In your downstream Terraform configuration, add the `module` block to reference the ARC module(s) in your `main.tf` file.
+* Set the `source` of the `module` block to point to https://github.com/sourcefuse/terraform-aws-ref* (select an available module in GitHub).
+* Set the version of the upstream module to prevent upstream configuration drift between major and minor changes. 
+
+**Example**:
+
+```hcl
+module "my_arc_module" {
+  source = "git::https://github.com/sourcefuse/terraform-aws-refarch-module-name?ref=0.1.0"
+
+  # Set other module-specific variables here
+}
+```
+* `source` attribute: This specifies the GitHub repository URL.
+* `ref` parameter: This denotes the version of the module to be used. Replace `0.1.0` with your desired version tag from upstream.
+
+### Open-Source modules 
+All Open-Source ARC Modules are available on our [GitHub Repository](https://github.com/sourcefuse). 
+For our Terraform specific modules, see [arc-docs](https://sourcefuse.github.io/arc-docs/arc-iac-docs/).
+
+### Client Virtual Private Network (VPN) for your VPC
```

Comment:
Can the sections be re-ordered to it flows through things like they would if we were setting them up for real?
* Landing Zone/AFT
* Client VPN
* Bootstrapping
* Network (if not deployed with the LZA/AFT)
* DB
* etc